### DISCUSSION TITLE
Tessie-like direct telemetry

### DISCUSSION THREAD

**Discussion by Kacper2Kaczmarek** [teslamate-org/teslamate#3746] [Ideas]
Recently, a new telemetry method has been added to Tessie which communicates directly with the car and allows to monitor battery temperature, current, voltage, etc.: https://help.tessie.com/article/128-direct-telemetry

This would give us a lot of interesting data to look at. Is it a feature that could be implemented in Teslamate at some point in the future?

**Comment by KrzysztofHajdamowicz** [comment]
Teslemetry allows to use new fleet API by acting as reverse proxy, that can be leveraged.
